ironfish/src/wallet/accounts.test.ts

```diff
@@ -341,4 +342,146 @@ describe('Accounts', () => {
       )
     })
   })
+
+  describe('getTransactionStatus', () => {
+    it('should show unconfirmed transactions as unconfirmed', async () => {
+      const { node } = nodeTest
+
+      const accountA = await useAccountFixture(node.accounts, 'a')
+
+      const blockA1 = await useMinerBlockFixture(node.chain, undefined, accountA, node.accounts)
+      await expect(node.chain).toAddBlock(blockA1)
+
+      await node.accounts.updateHead()
+
+      const transaction = blockA1.minersFee
+
+      const transactionValue = await accountA.getTransaction(transaction.hash())
+      Assert.isNotUndefined(transactionValue)
+
+      const transactionStatus = await node.accounts.getTransactionStatus(
+        accountA,
+        transactionValue,
+      )
+
+      expect(transactionStatus).toEqual('confirmed')
```

Comment:
should this equal ('unconfirmed')?!

Reply:
It should! Looks like this test is broken!